iso/Dockerfile

```diff
@@ -20,6 +20,10 @@ FROM boot2docker/boot2docker:1.12.3
 RUN echo "1.12.3" > $ROOTFS/etc/version
 ENV DOCKER_SHA a3bb88649bcc47775c29dab7d95709f4dda04d0e
 
+# Add the /etc/os-release file
+ADD os-release $ROOTFS/etc
+RUN sed -i '/os-release/,/^EOOS/d' /make_iso.sh
```

Comment:
+1, this is required but then better to remove from b2dverion till EOOS because those variable are only used for the os-release info. Something like below.

`RUN sed -i '/^b2dVersion.*/,/^EOOS/d' /make_iso.sh`